loadtest/utils.go

```diff
@@ -87,3 +89,24 @@ func nextPowerOf2(val int) int {
 	val++
 	return val
 }
+
+// RandomFutureTime returns a random Unix timestamp, in milliseconds, in the interval
+// [now+deltaStart, now+deltaStart+maxUntil]
+func RandomFutureTime(deltaStart, maxUntil time.Duration) int64 {
```

Comment:
I'd find it more versatile if we returned a `time.Time` and then let the caller format it as needed. Otherwise, if we can only foresee a timestamp usage, we could rename this to reflect its behavior better.

Reply:
There's just one use case of this function right now and I don't want to spend excessive time refactoring this to accommodate all possible cases. This is fine for now can can be updated later as needed. I've already refactored this twice.